pyemma/msm/estimators/maximum_likelihood_msm.py

```diff
@@ -207,103 +155,7 @@ def estimate(self, dtrajs, **parms):
         MSM : :class:`pyemma.msm.MaximumlikelihoodMSM`
 
         """
-        return super(MaximumLikelihoodMSM, self).estimate(dtrajs, **parms)
-
-    def _estimate(self, dtrajs):
```

Comment:
I think parts of this estimate routine are shared. Can you split into general and specific parts and put the general parts here?